lib/tasks/factorybot.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+desc "Run Factroybot linter"
+task factorybot_lint: :environment do
+  if Rails.env.test?
+    DatabaseCleaner.clean_with(:deletion)
+    DatabaseCleaner.cleaning do
```

Comment:
It makes sense why you're using DatabaseCleaner - nice that it was explained in the commit ⭐️ 

However I do wonder if we can avoid the need for the gem by running this in a transaction, we can have a go at that together if you like?